static/src/javascripts/lib/closest.spec.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import closest from './closest';
+
+const closestProto = Element.prototype.closest;
+let nestedChild;
+
+beforeAll(() => {
+    // Remove the closest method from Element so we test the polyfill
+    delete Element.prototype.closest;
+
+    if (document.body) {
+        document.body.innerHTML = `
+        <div class="greatgrandparent">
+            <div class="greatgrandparent findthisone">
+                <div class="grandparent">
+                    <div class="parent">
+                        <div class="child" id="child">child content</div>
+                    </div>
+                </div>
+            </div>
+        </div>`;
+    }
+
+    nestedChild = document.querySelector('.child');
+});
+
+afterAll(() => {
+    // Flow moans that I'm assigning something to a prototype
+    // $FlowFixMe
```

Comment:
Flow says 

```js
static/src/javascripts/lib/closest.spec.js:15
 15:     Element.prototype.closest = closestProto;
                           ^^^^^^^ property `closest`. Covariant property `closest` incompatible with contravariant use in
 15:     Element.prototype.closest = closestProto;
         ^^^^^^^^^^^^^^^^^^^^^^^^^ assignment of property `closest`
```

Can anyone explain what I might need to do? I read some stuff on covariance and cotravariant but my brain can't quite apply it to this...